cli/command_repository_upgrade.go

```diff
@@ -72,9 +72,24 @@ func (c *commandRepositoryUpgrade) setup(svc advancedAppServices, parent command
 
 	rollbackCmd.Action(svc.directRepositoryWriteAction(c.forceRollbackAction))
 
+	validateCmd := parent.Command("validate", "Validate the upgraded indexes.")
+	validateCmd.Flag("force", "Force rollback the repository upgrade, this action can cause repository corruption").BoolVar(&c.forceRollback)
```

Comment:
I'm not sure I understand how `--force` relates to its description. How about `--rollback-on-failure` instead?

Reply:
From our offline conversation, I understand now that when `--force` is specified, any validation errors will be ignored and the repository unlocked. The description of the command above does not match up with that and needs to be reworded.

Reply:
@adowair the code is a proof of concept for index comparison - command line wasn't really considered for this PR.